files/build_templates/buffers_config.j2

```diff
@@ -302,7 +302,7 @@ def
     "LOSSLESS_TRAFFIC_PATTERN": {
         "AZURE": {
             "mtu": "1024",
-            "small_packet_percentage": "100"
+            "small_packet_percentage": "50"
```

Comment:
Is this parameter used only for dynamic buffer calculations of XOFF and XON values?

Reply:
@kperumalbfn yes. By xoff only.